schemas/rum/action-schema.json

```diff
@@ -191,6 +191,19 @@
                       "readOnly": true
                     }
                   }
+                },
+                "name_source": {
+                  "type": "string",
+                  "description": "Action name computing source",
```

Comment:
> [!NOTE]
> Can we make the description a bit clearer as to what this attribute is for ? 
> Also was there any investigation of what enum values might be needed in the Mobile SDKs ?

Reply:
Yes. I will add a clearer description. The enum types are discussed in this [RFC](https://docs.google.com/document/d/1JMhuwp3As7KJLC3NwWKyiB7R655d8yPXLAdhr81kwq8/edit?tab=t.0). This feature is experimental. We could also leave it a non-enum string as it is `readOnly: false`. Wdyt?

Reply:
I think it maks sense to have an enum, I'll ask Mobile engineers working on SR privacy to take a look and add any missing value we might need.